modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/links/Link.java

```diff
@@ -38,28 +36,45 @@
 @Inherited
 public @interface Link {
   /**
-   * 
+   * The name of this link.
+   * @return the link's name
    **/
   String name() default "";
 
   /**
-   * 
+   * A relative or absolute reference to an OAS operation. This field is mutually exclusive of the operationId field, and must point to an Operation Object. Relative operationRef values may be used to locate an existing Operation Object in the OpenAPI definition.
+   * @return an operation reference
    **/
   String operationRef() default "";
 
   /**
-   * 
+   * The name of an existing, resolvable OAS operation, as defined with a unique operationId. This field is mutually exclusive of the operationRef field.
+   * @return an operation ID
    **/
   String operationId() default "";
 
   /**
-   * 
+   * Array of parameters to pass to an operation as specified with operationId or identified via operationRef.
+   * @return the list of parameters for this link
    **/
-  LinkParameters parameters() default @LinkParameters();
+  LinkParameter[] parameters() default {};
```

Comment:
Need to see the effect of this, very different than using a default of `@LinkParameters`